.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+    - name: 📃 Documentation Issue
+      url: https://github.com/reactjs/reactjs.org/issues/new
+      about: This issue tracker is not for documentation issues. Please file documentation issues here.
+    - name: 🤔 Questions and Help
+      url: https://stackoverflow.com/questions/tagged/react
```

Comment:
Let's change this URL to:

`https://reactjs.org/community/support.html`

Reply:
@necolas Reflected this too. c1459ca